Description: 
  With Python >= 2.5, 3rd argument of PyString_AsStringAndSize is Py_ssize_t*,
  not int *. Fix this type mismatch.
Author: Mike O'Connor <stew@vireo.org>
Bug-Debian: http://bugs.debian.org/476169
Last-Update: 2010-07-07

--- a/src/application/pythonscript.cpp
+++ b/src/application/pythonscript.cpp
@@ -99,7 +99,7 @@
 			if ( !(py_traceback=cStringIO->cgetvalue(catch_stderr)) )
 				throw ApplicationError("unable to get traceback","PythonScript::run()");
 			
-			int length;
+			Py_ssize_t length;
 			char *traceback;
 			if (PyString_AsStringAndSize(py_traceback, &traceback, &length))
 				throw ApplicationError("unable to convert traceback to char*","PythonScript::run()");
